lib/src/svg/xml_parsers.dart

```diff
@@ -125,19 +125,19 @@ PaintServer parseLinearGradient(XmlElement el) {
 
   parseStops(stops, colors, offsets);
 
+  final Matrix4 transform = parseTransform(getAttribute(el, 'gradientTransform', def: null));
+  
   return (Rect bounds) {
-    final Offset from = new Offset(
-      bounds.left + (bounds.width * x1),
-      bounds.left + (bounds.height * y1),
-    );
-    final Offset to = new Offset(
-      bounds.left + (bounds.width * x2),
-      bounds.left + (bounds.height * y2),
-    );
+    final from = Vector3(bounds.left + x1, bounds.top + y1, 0.0);
+    final to = Vector3(bounds.left + x2, bounds.top + y2, 0.0);
+    if(transform != null){
+      transform.transform3(from);
```

Comment:
Should be `from = transform.transform3(from);`

Reply:
they are the same, transform3 alters the parameter.

Reply:
Ahhh - I'm just used to getting bitten by similarly named methods that don't do this ;)